state.go

```diff
@@ -50,7 +55,8 @@ type KatzenmintState struct {
 	validators       map[string]pc.PublicKey
 	validatorUpdates []abcitypes.ValidatorUpdate
 
-	deferCommit []func()
+	deferCommit     []func()
+	prevCommitError error
```

Comment:
Are we going to use `prevCommitError`, eg: print?

Reply:
Its meant to avoid the app reporting the same error when commit fails over a period